tasks/structs.go

```diff
@@ -16,29 +16,35 @@ package tasks
 
 //go:generate go-enum -f=structs.go
 
-// TaskType x ENUM(
-// Deploy,
-// UnDeploy,
-// ScaleOut,
-// ScaleIn,
-// Purge,
-// CustomCommand,
-// CustomWorkflow,
-// Query,
-// Action
-// ForcePurge
-// AddNodes
-// RemoveNodes
-// )
+// TaskType is an enumerated type for tasks
+/*
+ENUM(
+Deploy
+UnDeploy
+ScaleOut
+ScaleIn
+Purge
+CustomCommand
+CustomWorkflow
+Query
+Action
+ForcePurge // ForcePurge is deprecated and should not be used anymore this stay here to prevent task renumbering colision
```

Comment:
ForcePurge is still used in rest/deployments.go line 310 in deleteDeploymentHandler().
Should it be replaced by a warning there as well ?

Reply:
Well, you are right I did not see this.
But it seems to be a valid use case here for force purge... WDYT?
This should be refactored later as we discussed IMHO.

Reply:
Yes, ok !